storages/domain_style.go

```diff
@@ -0,0 +1,41 @@
+package storages
+
+import (
+	"net/http"
+	"fmt"
+	"github.com/allegro/akubra/log"
+)
+
+const (
+	// InternalHostHeader is used for rewriting domain style
+	InternalHostHeader   = "X-Akubra-Internal-Host-3yeLjyjQNx"
+	// InternalBucketHeader used for rewriting domain style
+	InternalBucketHeader = "X-Akubra-Internal-Bucket-lejK0EpVZy"
+	// PathStyleFormat is a S3 path style format
+	PathStyleFormat      = "/%s%s"
+)
+
+type domainStyleInterceptor struct {
+	roundTripper http.RoundTripper
```

Comment:
`roundTripper` is't necessary in type ... struct, I think

Reply:
`domainStyleInterceptor` is just a wrapper for a `RoundTripper`. It does something with the request (check the `RoundTrip` function below) and delegates to the original `RoundTripper` that you provided in the constructor